internal/services/notification/db/migrate.go

```diff
@@ -69,3 +74,77 @@ func (d *DB) migrateV3(tx *sql.Tx) error {
 
 	return nil
 }
+
+func (d *DB) migrateV4(tx *sql.Tx) error {
+	switch d.sdb.Type() {
```

Comment:
Why don't use use the same logic for both db types (instead of json functions for postgres)?

Is a temporary table really needed? Can't you just add a nullable column, add the data and then make it non nullable?

Reply:
here https://www.techonthenet.com/sqlite/tables/alter_table.php#:~:text=Modify%20column%20in%20table,data%20into%20the%20new%20table. describe that is not possible use ALTER TABLE to add nullable with sqlite..  I tried with `ALTER TABLE new_runwebhook MODIFY project_id varchar NOT NULL` but not work.

Reply:
I meant for postgresql. For sqlite we already have migrations, that creates a new table and copies data to the new table. So you can unify the import logic from the json data to the new column.